src/server.js

```diff
@@ -0,0 +1,52 @@
+const express = require("express");
+const { getCollection } = require("./database");
+const { ObjectID } = require("mongodb");
+const cors = require("cors");
+const api = express();
+api.use(
+  cors({
+    origin: "http://localhost:3000"
+  })
+);
+
+const port = 8080;
+
+api.get("/pastes/:id", async (request, response) => {
+  try {
+    response.writeHead(200, { "Content-Type": "text/json" });
+    const pasteId = await get(request.params.id);
+    return response.end(pasteId);
+  } catch (error) {
+    return response.end("Error");
+  }
+});
+api.post("/pastes", async (request, response) => {
+  try {
+    let body = "";
+    request.on("data", function(data) {
+      body += data;
+    });
+    request.on("end", async function() {
+      response.writeHead(200, { "Content-Type": "text/json" });
```

Comment:
I think it should be `application/json` instead of `text/json`. See https://stackoverflow.com/questions/477816/what-is-the-correct-json-content-type